docs/getting_started.md

```diff
@@ -0,0 +1,96 @@
+# Getting Started with the AI Reference Implementation
+
+Welcome to the AI Reference Implementation! This guide will walk you through the steps to get started with deploying an AI environment on Azure using Terraform. By following these instructions, you can quickly set up a secure, scalable, and compliant AI platform tailored to your specific needs.
+
+***
+
+## Step 1: Assess Your AI Scenario
+
+Before you begin, it’s essential to understand the specific AI scenario you’re working on. The AI Reference Implementation supports a variety of use cases, such as Retrieval-Augmented Generation (RAG), Speech-to-Text call summarization, and more. Each scenario may require different configurations and resources.
+
+1. **Identify Your Scenario:** Determine the AI scenario you want to implement. This will guide the selection of modules and configuration options.
+2. **Explore Examples:** Check the `examples/` folder in this repository for pre-configured examples that match your scenario. These examples provide a starting point and demonstrate how to configure the AI Reference Implementation for specific use cases.
+
+* * *
+
+## Step 2: Set Up Prerequisites
+
+To use the AI Reference Implementation, you'll need to set up your development environment with the required tools and dependencies.
+
+1. **Install Required Tools:** Ensure you have the necessary tools, such as Terraform and Azure CLI, installed on your local machine. Detailed instructions for setting up your environment can be found in the Environment Setup document.
+    
+2. **Authenticate with Azure:** Follow the instructions in the Environment Setup guide to authenticate with Azure and ensure Terraform can manage your resources.
+    
+
+* * *
+
+## Step 3: Create a Terraform Project
+
+Now that you have your prerequisites in place, it’s time to create a Terraform project that references the AI Reference Implementation module.
+
+1. **Create a New Directory:** Start by creating a new directory for your Terraform project.
+    
+    ```bash
+    mkdir my-ai-project
+    cd my-ai-project
+    ```
+    
+2. **Initialize a Terraform Configuration File:** Create a `main.tf` file in your project directory and reference the AI Reference Implementation module.
+    
+    ```hcl
+    module "ai_reference_implementation" {
+      source  = "Azure/avm-ptn-ai-reference-implementation/azurerm"
+      version = "x.x.x"
```

Comment:
We would be able to populate the version and to add the link when we do have a version that is published in the terraform repository for now we don't hve the link and the version so this would not be there. 

About the closing the repo in the getting started the instructions are to create a new repo not to clone the repo. THe instructions to clone the repo are in the contributing to AIRI.